### DISCUSSION TITLE
Does this support `.AppInstaller` files?

### DISCUSSION THREAD

**Discussion by beedell-roke** [russellbanks/Komac#970] [Q&A] [answered]
[`microsoft/winget-pkgs/issues/199194#issuecomment-2548569634`](https://github.com/microsoft/winget-pkgs/issues/199194#issuecomment-2548569634) appears to state that it does not, despite (presumably) supporting `.AppX`, `.MSIX`, and `.MSIXBundle`. If so, is it uniquely difficult to support?

**Comment by russellbanks** [comment, marked as answer]
komac currently doesn't support `.AppInstaller` files. This is just because I've never come across them before and currently, I don't believe there are any `.appinstaller` files in [winget-pkgs](https://github.com/search?q=repo%3Amicrosoft%2Fwinget-pkgs%20.appinstaller&type=code). If WinGet does in fact support `.AppInstaller` files then it should be easy for me to add support for it to komac.

**Reply by beedell-roke** [reply]
> https://github.com/russellbanks/Komac/discussions/970#discussioncomment-11596998

@russellbanks, good point - I don't see any at [`github.com/search?q=repo%3Amicrosoft%2Fwinget-pkgs+.appinstaller&type=code`](https://github.com/search?q=repo%3Amicrosoft%2Fwinget-pkgs+.appinstaller&type=code&p=1), and https://github.com/microsoft/winget-pkgs/issues/166364#issuecomment-2273528233 seems to fairly explicitly state that they're undesirable for WinGet's primary use case. Thanks.
